vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -154,6 +138,20 @@ def setup_continuous_delivery(self, swap_with_slot, app_type_details, cd_project
         else:
             raise RuntimeError('Unknown status returned from provisioning_configuration: ' + response.ci_configuration.result.status)
     
+    def create_vsts_account(self, creds, vsts_account_name):
+        aex_url = 'https://app.vsaex.visualstudio.com'
+        accountClient = Account('4.0-preview.1', aex_url, creds)
+        self._update_progress(0, 100, 'Creating or getting Team Services account information')            
+        regions = accountClient.regions()
+        if regions.count == 0:
+            raise RuntimeError('Region details not found.')
+        region_name = regions.value[0].name
+        create_account_reponse = accountClient.create_account(vsts_account_name, region_name)
+        if create_account_reponse.id:
```

Comment:
you are not checking whether account_exists or not any longer. is that not really helpful?

Reply:
It will increase 1 extra REST call, anyways create_collection is doing it for us.
Also, we did the same in PE Service as well.

Reply:
if api is checking for existence and throwing proper error, then it is fine